nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1059,6 +1117,9 @@ in
           '' + optionalString (i.mtu != null) ''
             echo "setting MTU to ${toString i.mtu}..."
             ip link set "${i.name}" mtu "${toString i.mtu}"
+          '' + ''
+            echo -n "bringing up interface... "
+            ip link set "${i.name}" up && echo "done" || (echo "failed"; exit 1)
```

Comment:
Do you recall why this is necessary? It should be fine but just interested. :)

Reply:
```exit 1``` inside () means exit of ()-block only

Reply:
Ah, I meant the bringing up as a whole but didn't see that this was removed from the addresses service. This is fine! :+1: